packages/gestalt/src/TypeaheadInputField.js

```diff
@@ -0,0 +1,169 @@
+// @flow strict
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import layout from './Layout.css';
+import styles from './TypeaheadInputField.css';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import FormLabel from './FormLabel.js';
+
+type Props = {|
+  accessibilityLabel: string,
```

Comment:
@simkessy let's remove `accessibilityLabel` and keep `label` - they currently have overlapping behavior